features/address.js

```diff
@@ -14,10 +14,10 @@ const AddressTypes = {
   'POINTER': 0b0100,
   'ENTERPRISE': 0b0110,
   'BOOTSTRAP': 0b1000,
-  'REWARDS': 0b1110
+  'REWARD': 0b1110
```

Comment:
I renamed this so its consistent with the `packRewadAddress` function, also in other places.